### PR TITLE
README Update

### PR DESCRIPTION
**Pull request by ardacebi**
Edited text for a better visual experience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-TinyWebDB
+# Makeroid TinyWebDB Server
+The official Makeroid TinyWebDB Server for you to use only for test purposes.
```
Comment (barreeeiroo) [inline README.md]: Technically is for both testing and production purposes, as to set up one is not an easy action and users won't have time to do so

**Review by barreeeiroo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ardacebi) [reply README.md]: So what do you suggest me to do? Change the phrase from:
**"The official Makeroid TinyWebDB Server for you to use only for test purposes."** 
to
**"The official Makeroid TinyWebDB Server for you to use only for test and production purposes."?**

**Comment by ardacebi** [comment]
@barreeeiroo is it ok now?

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-TinyWebDB
+# Makeroid TinyWebDB Server
+The official Makeroid TinyWebDB Server for you to use only for test and production purposes.
```
Comment (barreeeiroo) [inline README.md]: Just leave the second line as _The official Makeroid TinyWebDB Server for you_

**Review by barreeeiroo — CHANGES_REQUESTED** [review]

**Comment by ardacebi** [comment]
It now looks more clean.

**Review by barreeeiroo — APPROVED** [review]

**Pull request merged**